db/seed.sql

```diff
@@ -4262,7 +4259,7 @@ CREATE TABLE `eav_attribute_group` (
 
 LOCK TABLES `eav_attribute_group` WRITE;
 /*!40000 ALTER TABLE `eav_attribute_group` DISABLE KEYS */;
-INSERT INTO `eav_attribute_group` VALUES (1,1,'General',1,1,'general',NULL),(2,2,'General',1,1,'general',NULL),(3,3,'General',10,1,'general',NULL),(4,3,'General Information',2,0,'general-information',NULL),(5,3,'Display Settings',20,0,'display-settings',NULL),(6,3,'Custom Design',30,0,'custom-design',NULL),(7,4,'Product Details',10,1,'product-details','basic'),(8,4,'Advanced Pricing',40,0,'advanced-pricing','advanced'),(9,4,'Search Engine Optimization',30,0,'search-engine-optimization','basic'),(10,4,'Images and Videos',20,0,'image-management','basic'),(11,4,'Design',50,0,'design','advanced'),(12,4,'Autosettings',60,0,'autosettings','advanced'),(13,5,'General',1,1,'general',NULL),(14,6,'General',1,1,'general',NULL),(15,7,'General',1,1,'general',NULL),(16,8,'General',1,1,'general',NULL);
+INSERT INTO `eav_attribute_group` VALUES (1,1,'General',1,1,'general',NULL),(2,2,'General',1,1,'general',NULL),(3,3,'General',10,1,'general',NULL),(4,3,'General Information',2,0,'general-information',NULL),(5,3,'Display Settings',20,0,'display-settings',NULL),(6,3,'Custom Design',30,0,'custom-design',NULL),(7,4,'Product Details',1,1,'product-details','basic'),(8,4,'Advanced Pricing',4,0,'advanced-pricing','advanced'),(9,4,'Search Engine Optimization',3,0,'search-engine-optimization','basic'),(10,4,'Images and Videos',2,0,'image-management','basic'),(11,4,'Design',5,0,'design','advanced'),(12,4,'Autosettings',6,0,'autosettings','advanced'),(13,5,'General',1,1,'general',NULL),(14,6,'General',1,1,'general',NULL),(15,7,'General',1,1,'general',NULL),(16,8,'General',1,1,'general',NULL);
```

Comment:
This is only changes to the `sort_order` column, not even sure why this happened.